oval/util.go

```diff
@@ -438,7 +438,8 @@ func lessThan(family, newVer string, packInOVAL ovalmodels.Package) (bool, error
 	case constant.RedHat,
 		constant.CentOS,
 		constant.Alma,
-		constant.Rocky:
+		constant.Rocky,
+		constant.Fedora:
```

Comment:
Fedora does not reuse RHEL's OVAL, so this process is not necessary; you should refer to Oracle or Amazon to write it.